### ISSUE TITLE
Push new version to npm

### ISSUE THREAD

**Issue by jgeurts** [bwin/grunt-asar#5]
Any chance of pushing the version from the `update-asar` branch to npm?

**Comment by TemaSM** [comment]
+1
Someone still maintains this repo?

> Tip for everyone, who wants to install branch `update-asar` using npm:
> `npm i bwin/grunt-asar#update-asar`

**Comment by bwin** [comment]
Yes I do. Sorry for abandoning it for so long. I'm waiting for atom/asar#57.

**Comment by TemaSM** [comment]
@bwin Ahh, so nice to hear it! Good luck with dev and big thanks for what you are doing! :wink:

**Comment by RossComputerGuy** [comment]
@bwin Hey that PR was merged in Feb 17, 2016.
